leits/MeetingBar
# ✨feature: new visibility option for non all day events

### Status
**READY**

### Description
We can now better control visibility of non all day events - following options are available:

- standard is show which alls all events, regardless the event contains a meeting or a link

![image](https://user-images.githubusercontent.com/3872101/114938985-1574be00-9e40-11eb-9770-9b01214a3400.png)


- show as inactive without any link hides events in the statusbar when the event does not contain any link, but shows in the menubar as inactive - links in events now will also open in the browser

![image](https://user-images.githubusercontent.com/3872101/114938943-0857cf00-9e40-11eb-9101-47b283bcad63.png)



- show as inactive without any meeting hides events in the statusbar when the event does not contain any known meeting link, but shows in the menubar as inactive

![image](https://user-images.githubusercontent.com/3872101/114938889-f413d200-9e3f-11eb-9a46-6639ae0fa28f.png)


- hide without any link hides events in the statusbar and menubar when the event does not contain any link

![image](https://user-images.githubusercontent.com/3872101/114939026-258c9d80-9e40-11eb-9878-621db05ded7a.png)


- hide without any meeting link hides events in the statusbar and menubar when the event does not contain any known meeting link

![image](https://user-images.githubusercontent.com/3872101/114939060-33422300-9e40-11eb-8ef5-0e9cf5790029.png)


### Steps to Test or Reproduce
- Create 3 events (1 without any link, 1 with a known meeting service, 1 with a valid link, but no known meeting service)
- test the different options and check the result in menubar and statusbar
